packages/plugin-react-native-navigation/README.md

```diff
@@ -0,0 +1,11 @@
+# `@bugsnag/plugin-react-native-navigation-performance`
+
+> TODO: description
```

Comment:
Can we put some brief content in here, but link to online docs rather than include any detailed content.

Reply:
I see we have one for React Navigation, so look to mimic that.